blogs/8_berkeley_function_calling_leaderboard.html

```diff
@@ -1022,7 +1029,8 @@ <h3 id="cost">Cost & Latency</h3>
                                             <img src="../assets/img/blog_post_8_gpu_formula.jpg" width="90%">
                                             We use the Azure ND40rs-v2 instance (8X V100 GPU) April 2024 pay-as-you-go
                                             pricing in the
-                                            cost calculation. This is not meant to be precise as price change all the time, but to give an idea of
+                                            cost calculation. This is not meant to be precise as price change all the
```

Comment:
"This is not drift to be precise as the price can change often. We will try our very best to keep this up-to-date on daily or at least weakly basis. Nonetheless this should give an idea of what the magnitude of costs should look like, and help understand the relative ordering all things constant."

Reply:
addressed